Sources/Redis/Request+Redis.swift

```diff
@@ -1,12 +1,14 @@
 import Vapor
 
 extension Request {
-    public var redis: Redis {
-        .init(request: self)
-    }
-
     public struct Redis {
+        let redisID: RedisID
```

Comment:
I feel this should be public on both this `Redis` and `Application.Redis` and named `id`